unlock-app/src/__tests__/services/web3Service.test.js

```diff
@@ -823,7 +823,10 @@ describe('Web3Service', () => {
       it('should emit a new transaction once it has been submitted', done => {
         expect.assertions(2)
         web3Service.sendTransaction = jest.fn((transaction, args, cb) => {
-          return cb(null, { event: 'transactionHash', args: { hash: '0x123' } })
```

Comment:
Same here. This should be fixed once https://github.com/unlock-protocol/unlock/pull/732 has been merged. Could you please rebase after that? Thanks!